tasks/aquatone.yml

```diff
@@ -0,0 +1,34 @@
+---
+# tasks file for aquatone
+
+# This tool is installed a bit differently.  The release zip for it
+# contains pre-compiled Go code, which is what we want to install.
+- name: Install aquatone
+  block:
+    - name: Install dependencies for aquatone
+      package:
+        name:
+          - chromium
+          - unzip
+    - name: Create a directory where aquatone will be installed manually
+      file:
+        group: "{{ group }}"
+        # Allow both root and the desired group to have full
+        # permissions in these directories
+        mode: 0775
+        path: /tools/{{ item.dir_name }}
+        state: directory
+      loop:
+        - {dir_name: "aquatone", repo: "https://github.com/michenriksen/aquatone/releases/download/v1.7.0/aquatone_linux_amd64_1.7.0.zip"}
```

Comment:
Can we make this repo URL (or maybe just the version #) into a variable?

Reply:
Yes.  See commit 19cd876.

Reply:
👍